hs_core/hydroshare/resource.py

```diff
@@ -747,6 +749,7 @@ def publish_resource(pk):
     resource.raccess.immutable = True
     resource.raccess.save()
     resource.doi = "to be assigned"
+    resource.metadata.create_element('date', type='published', start_date=now().isoformat())
```

Comment:
@pkdash Can you remove the create_element() statement line above? I am working on resource publication and have rewritten this function publish_resource() and put this create_element() call into resource_modified() function, which is more appropriate. I should be able to create a PR sometime next week for code review.